src/components/searchbar/SearchBar.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+import { UilSearch, UilLocationPoint } from "@iconscout/react-unicons";
+
+function SearchBar() {
+  return (
+    <>
+      <Search type="text" placeholder="search for a city..." />
+      <UilSearch size={25} />
+      <UilLocationPoint size={25} />
+    </>
+  );
+}
```

Comment:
Has the `SearchBar` component any functionality? Just asking.

Reply:
Not now going to add the functionality later